docs/snippets/myst/card-link.txt

```diff
@@ -1,15 +1,25 @@
 (cards-clickable)=
-## Cards Clickable
+
+## Clickable cards
+
+Using the `link` and `link-type` options, you can turn an entire card into a clickable link. Try hovering over then clicking on the cards below:
```

Comment:
I changed the snippets to match the entire "Clickable cards" section (minus the `literalinclude` blocks).

I wanted to also match the line breaks but this produced a discrepancy in the printed doc trees under `test_snippets/`.

For example, putting a break between "a" and "clickable", like so:

```
Using the `link` and `link-type` options, you can turn an entire card into a
clickable link. Try hovering over then clicking on the cards below:
```

produced the following output:

```
            <paragraph>
                Using the 
                <literal>
                    link
                 and 
                <literal>
                    link-type
                 options, you can turn an entire card into a
                
                clickable link. Try hovering over then clicking on the cards below:
```

Whereas the same line break in the rST file produced:

```
            <paragraph>
                Using the 
                <literal>
                    link
                 and 
                <literal>
                    link-type
                 options, you can turn an entire card into a
                clickable link. Try hovering over then clicking on the cards below:
```

Which is exactly the same except for that extra blank line between "a" and "clickable".

So to make the tests pass, I just put each paragraph in the MyST and rST snippets on a single unbroken line.

I tried using the [terminal backslash syntax](https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#line-breaks) in the MyST snippet to see if that would fix it, like so:

```
Using the `link` and `link-type` options, you can turn an entire card into a\
clickable link. Try hovering over then clicking on the cards below:
```

But this added a Docutils raw node that was not present in the rST-generated doc tree, like so:

```
                 options, you can turn an entire card into a
                <raw format="html" xml:space="preserve">
                    <br />
                <raw format="latex" xml:space="preserve">
                    \\
                clickable link. Try hovering over then clicking on the cards below:
```

At that point, I gave up. 🙃